chainerrl/envs/multiprocess_vector_env.py

```diff
@@ -53,6 +54,13 @@ class MultiprocessVectorEnv(chainerrl.env.VectorEnv):
     """
 
     def __init__(self, env_fns):
+        if np.__version__ == '1.16.0':
```

Comment:
In general, `numpy.lib.NumpyVersion` could be used.  Looks OK without it to detect a single version.

Reply:
I intend to detect a single version. I haven't check if pre-release versions of numpy have the same issue.